net-libs/libsrsirc/libsrsirc-0.0.14.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="A lightweight, cross-platform IRC library"
+HOMEPAGE="https://github.com/fstd/libsrsirc"
+SRC_URI="http://penenen.de/${P}.tar.gz"
+LICENSE="BSD"
+
+SLOT="0"
+KEYWORDS="~x86 ~amd64"
+IUSE="static-libs ssl"
+
+DEPEND="
+	ssl? ( dev-libs/openssl:0= )"
+RDEPEND="${DEPEND}"
+
+src_configure() {
+	use static-libs && econf $(use_with ssl)
+	! use static-libs && econf --disable-static $(use_with ssl)
```

Comment:
That would do `--enable-static` in case `static-libs` is set. Which is slightly different to what you asked for.